build.cake

```diff
@@ -19,7 +19,7 @@ var artifactsBinNet452Directory = artifactsBinDirectory.Combine("net452");
 var artifactsBinNetStandard15Directory = artifactsBinDirectory.Combine("netstandard1.5");
 var artifactsDocsDirectory = artifactsDirectory.Combine("docs");
 var artifactsDocsApiDocsDirectory = artifactsDocsDirectory.Combine("ApiDocs-" + gitVersion.LegacySemVer);
-var artifactsDocsRefDocsDirectory = artifactsDocsDirectory.Combine("RefDocs-" + gitVersion.LegacySemVer);
+var artifactsDocsRefDocsDirectory = artifactsDocsDirectory.Combine("RefDocs-" + gitVersion.LegacySemVer + "-html");
```

Comment:
@vincentkam How created folder(with `-html` suffix) will be processed in next steps?

Reply:
Hmm... doing a search for `artifactsDocsRefDocsDirectory `, I currently can't see any problem with changing the value of this variable. 
@MikalaiMazurenka Am I correct in assuming that you changed the file name so that directory that we create would have the same name as the unzipped zip file?
I think it does no harm to leave this line changed (without the `-html` suffix, but it also does no harm to leave it with the `-html` suffix, so I leave it to @MikalaiMazurenka as to which you'd prefer :)

Reply:
My question was more about how the created folder(no matter what name with) will be processed further? How this folder will be copied on the web server.

Reply:
@vincentkam yes, it works exactly this way. I'll probably go with "-html" way so the end result looks as in previous version. And I will update this PR to include apidocs task change.

Reply:
Ah sorry Dima, I misread your initial question: thanks for the clarification! The contents of the folder need to be manually dumped into our gh-pages branch. For more detail, see https://docs.google.com/document/d/1qezMwQWe2gAt4aRnYBh0VNc3JB6J93KIqyOoZlG5eKU/edit#heading=h.oh61nq1p11ty

Reply:
Ok, thanks! it's helpful :)

Reply:
Why are you adding "-html" to the folder name?

Reply:
Previously ONLY the zip file had "-html" in the file name.

I don't think it's useful to add "-html" any more.

Reply:
I think this suffix was added because in the previous implementation when we had unzipped a zip file (that was being created previously) without any additional zipping options, the folder which contained unzipped data had this suffix. 

So, now we just skip the intermediate step (zipping) and immediately create the previously resulted folder (with the same name as before).

Reply:
So, the question is why we had this suffix before? Because it looks like this suffix had no a lot of sense in the previous implementation too

Reply:
NOTE: I think the only thing why this suffix can be required is just to have implementation as close as possible to the previous implementation

Reply:
In general it is good to be as close as possible to the previous implementation... except when the previous implementation is no longer wanted.

I vote getting rid of the unnecessary "-html" suffix.

Reply:
Removed "-html" postfix. And made same changes to "apidocs" task